stablehlo/reference/Element.cpp

```diff
@@ -538,6 +564,38 @@ Element min(const Element &e1, const Element &e2) {
       });
 }
 
+Element power(const Element &e1, const Element &e2) {
+  Type type = e1.getType();
+
+  if (isSupportedIntegerType(type)) {
+    APInt base = e1.getIntegerValue();
+    APInt exponent = e2.getIntegerValue();
+    bool isSigned = base.isSignedIntN(base.getBitWidth());
```

Comment:
This is not going to do what you expect, because `APInt::isSignedIntN` already assumes that the number is signed. `isSupportedSignedIntegerType(type)` is the right way of checking this.